mentalmodel.md

```diff
@@ -0,0 +1,94 @@
+  datetime --- time
+  date --- yearmonth
+  date --- monthday
+```
+![Mental Model Diagram](./objectrelation.png)
+
+Before continuing I'd also like to call out the following API principles to keep in mind when evaluating the proposal:
+
+1. All objects are immutable
+2. Methods beginning with `with` will result in objects that contain equal or more data and are therefore safe in terms of lossing information.
+3. Methods beginning with `get` will result in retrieving an aspect of the object. Thus the value retrieved will have less information that the original object.
+4. The calculation methods `plus`, `minus` and `difference` act with or result in objects of the same type. `difference` is only permissible with objects of the same information content, while `plus` and `minus` are permissible only with `Durations` (or objects containing the same level of information) generated using the `difference` method.
+
+**Please also see the detailed [API description of these temporal objects](objects.md)!!!**
+
+## Instant
+
+`Instant` objects are meant to represent a specific point in absolute time. For convenience we have chosen that point to be *midnight of January the 1st 1970 UTC* as our *epoch*. The choice fell upon that specific date and time due to the precendent of the [POSIX Epoch](http://pubs.opengroup.org/onlinepubs/9699919799/mindex.html) and Unix Epoch based on it.
+
+We decided to ignore  leap seconds since they are arbitrarily decided upon and unpredictable. An `Instant` can therefore not be said to be on the TAE timeline, but rather on the UTC timeline. In terms of leap-seconds we accept them a input (`fromString`) but immediately convert them to the relevant numeric representation and therefore ignoring the fact that this came from a leap-second.
+
+Because `Instant` is just a point in time, it has no concept of calendars or time-units as we use in civil society, the `Instant` objects do not have members representing these either.
+
+Conceptually `Instant`s are not geopgraphically tied to any place. They are equally valid on Mars and Venus as on earth.
+
+## OffsetDateTime
+
+`OffsetDateTime` objects represent a point in time together with a specific offset from UTC. Given that these offets are only valid on Earth, they are tied to a place. However they do not have a timezone or dayligh-davings adjustments or anything of that nature.
+
+As such they can be considered at best to be tied to the planet Earth rather than a specific locality. However because they are tied to the planet, they do have a concept of time units uch as hours and days. And because we restrict ourselves to the proleptic gregorian calendar, they do have months and years that correspond with our western understanding of these ideas.
+
+## ZonedDateTime
+
+`ZonedDateTime` objects represent a point in time in a specific IANA timezone. Since these timezones are tied to a specific locality (city/country), these are more precise and observe daylight savings changes.
```

Comment:
```suggestion
`ZonedDateTime` objects represent a point in time in a specific IANA timezone. Since these timezones are tied to a specific locality (city/state/country), these are more precise and observe daylight savings changes.
```